client/src/components/Entrance.js

```diff
@@ -1,7 +1,15 @@
-import "./styles/Entrance.css";
+import { Link } from "react-router-dom";
 
-function Entrance() {
-  return <div>Entrance</div>;
+export default function Entrance(props) {
```

Comment:
A good way to get props values is to get them directly. Like this for example:
function Entrance({ID, getNewTable}){ ... }